src/templates/base/2019/base.html

```diff
@@ -127,15 +125,13 @@
             <span class="menu-btn__bar"></span>
           </button>
           <ul class="menu">
-            <li><a href="{{ url_for('table_of_contents', year=year, lang=lang) }}">{{ self.table_of_contents_title() }}</a></li>
-            <li><a href="{{ url_for('contributors', year=year, lang=lang) }}">{{ self.contributors_title() }}</a></li>
-            <li><a href="{{ url_for('methodology', year=year, lang=lang) }}">{{ self.methodology_title() }}</a></li>
+            <li><a href="{{ url_for('table_of_contents', year=year, lang=lang) }}" class="toc">{{ self.table_of_contents_title() }}</a></li>
```

Comment:
Like I mentioned in another comment, I don't think we need classes on these links to hide them in CSS. I think the thing to do is overwrite this block in `2020/base.html`.

Reply:
And, as per other comment, I'm against creating a `2020/base.html` right now as see no reason (other than this - which is a short term, temporary fix). We can remove these later if we want once we either have a reason for `2020/base.html` or launch 2020 properly with these pages using 2019 templates.

Reply:
As per other comment, I figured out better way to do this in https://github.com/HTTPArchive/almanac.httparchive.org/commit/d5b2e678a96e63644b8697f213b4489e29805576 (and follow up in https://github.com/HTTPArchive/almanac.httparchive.org/pull/893/commits/a9baa2b564c14fa2b50c08f5ca5104956bac8f73)